dictionary.go

```diff
@@ -8,14 +8,33 @@ import (
 	"github.com/segmentio/parquet-go/deprecated"
 	"github.com/segmentio/parquet-go/encoding"
 	"github.com/segmentio/parquet-go/encoding/plain"
+	"github.com/segmentio/parquet-go/hashprobe"
 	"github.com/segmentio/parquet-go/internal/bitpack"
 	"github.com/segmentio/parquet-go/internal/unsafecast"
 )
 
 const (
-	// Completely arbitrary, feel free to adjust if a different value would be
-	// more representative of the map implementation in Go.
-	mapSizeOverheadPerItem = 8
+	// Maximum load of probing tables. This parameter configures the balance
+	// between memory density and compute time of probing operations.
+	//
+	// Smaller values result in lower collision probability when inserting
+	// values in probing tables, but also increase memory utilization.
+	//
+	// TODO: make this configurable by the application?
+	hashprobeTableMaxLoad = 0.9
```

Comment:
What is the range of acceptable values here? (0, 1) ?

Reply:
Yes